pkg/generators/openapi_test.go

```diff
@@ -131,6 +131,7 @@ type Blah struct {
 	WithStructTagExtension string `+"`"+`patchStrategy:"merge" patchMergeKey:"pmk"`+"`"+`
 	// a member with a list type
 	// +listType=atomic
+	// +default=["foo", "bar"]
```

Comment:
You tested the failure cases below, but I didn't see tests for the success cases like this one?